app/types/actions.ts

```diff
@@ -1,4 +1,4 @@
-export type ActionType = 'file' | 'shell';
+export type ActionType = 'file' | 'shell' | 'preview';
 
```

Comment:
`export type ActionType = 'file' | 'shell' | 'preview';` is this change required? i don't see it being used anywhere and relevant